src/users/dto/message.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { Message } from 'tonomy-id-sdk';
+
+export class MessageDto extends Message {}
+
+export class MessageRto {
+  message!: string;
```

Comment:
I thought this could be a string or object?

Reply:
the MessageDTO is the one the server Sends back and use
the MessageRTO is the string the server expects this is just for documentation for now